sources/packages/backend/libs/sims-db/src/entities/msfaa-number.model.ts

```diff
@@ -93,6 +93,7 @@ export class MSFAANumber extends RecordDataModel {
    * Student id related to this MSFAA.
    */
   @RelationId((msfaaNumber: MSFAANumber) => msfaaNumber.student)
```

Comment:
I think we can remove the `@RelationId` if we have now a `@Column`.

Reply:
Removed `@Column` and kept `@RelationId` as it is as per discussion with @andrewsignori-aot and @ann-aot